Slider/Assets/Scripts/Map/Mountain/AnimRedirect.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimRedirect : MonoBehaviour
```

Comment:
do u think u can change this to a more specific name

also isn't this literally just putting a transition arrow to the idle state in the animator (pls keep it for now tho i was thinking of adding more machines/pipes that might wanna be animated when this one finishes its anim)